Sources/BraintreeShopperInsights/BTPresentmentDetails.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+public enum BTExperimentType: String {
+    case test = "test"
+    case control = "control"
+}
```

Comment:
I think we can move this into it's own file and should be able to format the value here. Psudo code so may not compile as is but I think we can do something like:
```suggestion
public enum BTExperimentType: String {
    case test = "test"
    case control = "control"
    
    var formattedExperiment: String = """
            [
                { "exp_name" : "PaymentReady" },
                { "treatment_name" : self.stringValue }
            ]
            """
}
```
Then later when needed we can access and send to FPTI as `experimentType.formattedExperiment`

Reply:
ah ok I get it now. Thanks!